react-frontend/src/components/StyleComponents/card.js

```diff
@@ -230,3 +230,34 @@ export const IconCol = styled(Col).attrs({
   font-family: BC Sans, Noto Sans, Verdana, Arial, sans-serif;
   font-weight: 700;
 `;
+
+export const EventCardStyled = styled.div.attrs({
+  className: 'cardRound',
+})`
+  background-color: white;
+  border-radius: ${cardBorderRadius};
+  height: 100%;
+  margin-bottom: 20px;
+  width: 100%;
+  display: flex;
+  flex-direction: column;
+`;
+export const EventCardThumnail = styled.img`
```

Comment:
The images that get pulled from EventBrite that populate the card get stretched at some breakpoints:
![decent](https://user-images.githubusercontent.com/16946297/124504604-ab075600-dd7c-11eb-8133-70a1f2cac9e7.png)
![stretched](https://user-images.githubusercontent.com/16946297/124504616-afcc0a00-dd7c-11eb-8d31-5e58c9649219.png)

I found that using `object-fit: contain` is a potential satisfactory solution:
![object-fit-contain](https://user-images.githubusercontent.com/16946297/124504648-bf4b5300-dd7c-11eb-92eb-60d3bd9bf18a.png)